jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/params/BoosterParams.scala

```diff
@@ -244,7 +255,7 @@ private[spark] trait BoosterParams extends Params {
 
   setDefault(eta -> 0.3, gamma -> 0, maxDepth -> 6,
     minChildWeight -> 1, maxDeltaStep -> 0,
-    growPolicy -> "depthwise", maxBins -> 16,
+    growPolicy -> "depthwise", maxBins -> 16, batchsize -> (32 << 10),
```

Comment:
```batchsize``` -> ```batchSize```. There are some historical reasons, but new added params should follow correct convention.